### PR TITLE
Enable TorchBlade based on Torch-MLIR by default

### PR DESCRIPTION
**Pull request by tanyokwok**

### CODE REVIEW

File: pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp [hunk 1]
```diff
@@ -169,6 +169,42 @@ class ConvertAtenUnaryOp : public OpConversionPattern<AtenOpT> {
...
+    if (!inpTy)
+      return op.emitError("only RankedTensorType is supported");
+
+    auto outTy = OpConversionPattern<AtenOpT>::getTypeConverter()->convertType(
+        op.getType());
+    auto elemTy = inpTy.getElementType();
+
+    if (outTy != elemTy) {
+      auto output =
+          rewriter.create<tensor::ExtractOp>(op.getLoc(), elemTy, input);
```
Comment (qiuxiafei) [inline pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp:193]: A little bit confused ...  `tensor::ExtractOp` is usually created given a tensor `Value` and an indices `Value`s, that does the first argument `elemTy` mean here?

File: pytorch_blade/pytorch_blade/compiler/mlir/converters/BUILD [hunk 2]
```diff
@@ -34,28 +34,3 @@ cc_library(
     ],
     alwayslink = True,
 )
-
-cc_test(
-    name = "mhlo_converter_register_test",
```
[the remark this answers is not in the document]
Comment (tanyokwok) [reply pytorch_blade/pytorch_blade/compiler/mlir/converters/BUILD:39]: The old `MHLO` is ready to be moved after this pull request is merged. (I prefer to split the action into another PR.)

The test `mhlo_converter_register_test` is removed due to it failed when Torch-MLIR is enabled by default.

File: pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp [hunk 3]
```diff
@@ -57,6 +58,10 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       "aten::div",
       "aten::embedding",
       "aten::eq",
+      "aten::gt",
```
[the remark this answers is not in the document]
Comment (tanyokwok) [reply pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp:61]: This is white list for Torch-MLIR converters.

File: pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp (hunk 1 above)
Comment (tanyokwok) [reply pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp:193]: `elemTy` is given as the new output type. The `indices` parameter is optional.

File: pytorch_blade/scripts/build_pytorch_blade.sh [hunk 4]
```diff
@@ -58,12 +58,7 @@ function ci_build() {
     export TORCH_BLADE_DEBUG_LOG=ON
     # disable tf32 on A100
     export NVIDIA_TF32_OVERRIDE=0
-    pytest tests -v 2>&1 | tee -a py_test.out
-    TORCH_DISC_USE_TORCH_MLIR=true pytest tests/disc/ops/ -v -k \
-             "TestDiscActivation or TestDiscUnaryOps or TestDiscBinaryOps or \
-              TestDiscBroadcast or TestDiscReduction or TestDiscMatMul or \
-              TestDiscPermutation or TestDiscShapes or TestDiscSlices or \
-              TestDiscMemOps or TestConstOps or TestDiscLayerNorm or TestDiscNNOps"
+    TORCH_DISC_USE_TORCH_MLIR=true pytest tests -v 2>&1 | tee -a py_test.out
```
[the remark this answers is not in the document]
Comment (tanyokwok) [reply pytorch_blade/scripts/build_pytorch_blade.sh:61]: `TORCH_DISC_USE_TORCH_MLIR` has been already set to true by default, see `pytorch_blade/pytorch_blade/compiler/mlir/converters/mhlo_conversion.cpp`

The env setting here is used in the unittests, somewhere like `skipIfTorchMlirEnable`. I will remove all these usages after the old Mhlo converter is deprecated.

File: pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp [hunk 5]
```diff
@@ -169,6 +169,42 @@ class ConvertAtenUnaryOp : public OpConversionPattern<AtenOpT> {
...
+
+      if (elemTy.isIntOrIndex()) {
+        rewriter.replaceOpWithNewOp<arith::ExtSIOp>(op, outTy, output);
```
[the remark this answers is not in the document]
Comment (tanyokwok) [reply pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp]: done

File: pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp (hunk 3 above)
[the remark this answers is not in the document]
Comment (qiuxiafei) [reply pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp:61]: orz... my bad..

**Review by qiuxiafei — APPROVED** [review]
LGTM

**Pull request merged**